### PR TITLE
SPF/DKIM configuration changes now updates all accounts retroactively

### PR DESCRIPTION
**Pull request by euhmeuh**
Changing the global configuration of an exchange service did not reflect back to the accounts, only to the ones created after the change.

Now we call `updateFlagsOnAllAccounts` when applying the form.

### CODE REVIEW

File: src/resources/i18n/exchange/Messages_fr_FR.xml [hunk 1]
```diff
@@ -1163,8 +1163,8 @@ L'antispam Vade Secure est compatible avec le mode non autoritatif (uniquement s
    <translation id="exchange_ACTION_configure_services_label_minPasswordLength" qtlid="174828">Longueur minimale du mot de passe</translation>
    <translation id="exchange_ACTION_configure_services_label_complexityEnabled" qtlid="178003">Exigences de complexité</translation>
    <translation id="exchange_ACTION_configure_services_label_incoming_messages" qtlid="81871">Messages entrants</translation>
-   <translation id="exchange_ACTION_configure_services_label_spf" qtlid="402399">Activer la protection SPF</translation>
```
Comment (AxelPeter) [inline src/resources/i18n/exchange/Messages_fr_FR.xml]: I think you can keep the qtlid with theses changes. To confirm :)

Comment (FredericEspiau) [reply src/resources/i18n/exchange/Messages_fr_FR.xml]: I don't think so, the meaning of the sentence changed

File: src/exchange/configure/service-configure.controller.js [hunk 2]
```diff
@@ -80,6 +80,12 @@ angular
                     },
                     data: dataToSend
                 })
+                .then(() => this.services.APIExchange.post("/{organizationName}/service/{exchangeService}/updateFlagsOnAllAccounts", {
```
Comment (FredericEspiau) [inline src/exchange/configure/service-configure.controller.js]: Should be deported to a service

File: src/exchange/exchange.service.js [hunk 3]
```diff
@@ -196,6 +197,21 @@ angular
                 });
         }
 
+        setConfiguration (organization, serviceName, data) {
+            return this.services.APIExchange.put("/{organizationName}/service/{exchangeService}", {
```
Comment (jleveugle) [inline src/exchange/exchange.service.js]: We should begin using ovh-api-services imo :)

https://github.com/ovh-ux/ovh-api-services

File: src/resources/i18n/exchange/Messages_fr_FR.xml [hunk 4]
```diff
@@ -1163,8 +1163,8 @@ L'antispam Vade Secure est compatible avec le mode non autoritatif (uniquement s
...
-   <translation id="exchange_ACTION_configure_services_label_dkim" qtlid="402412">Activer la signature DKIM</translation>
+   <translation id="exchange_ACTION_configure_services_label_spf">Activer la vérification de la protection SPF</translation>
```
Comment (jleveugle) [inline src/resources/i18n/exchange/Messages_fr_FR.xml]: We should keep the qtlid here.

The meaning of the sentence won't change in the other language imo :)

**Pull request merged**